ui/components/app/gas-details-item/gas-details-item.js

```diff
@@ -39,14 +41,16 @@ const GasDetailsItem = ({ userAcknowledgedGasMissing = false }) => {
       detailTitle={<GasDetailsItemTitle />}
       detailTitleColor={COLORS.TEXT_DEFAULT}
       detailText={
-        <div className="gas-details-item__currency-container">
-          <LoadingHeartBeat estimateUsed={estimateUsed} />
-          <UserPreferencedCurrencyDisplay
-            type={SECONDARY}
-            value={hexMinimumTransactionFee}
-            hideLabel={Boolean(useNativeCurrencyAsPrimaryCurrency)}
-          />
-        </div>
+        useCurrencyRateCheck && (
+          <div className="gas-details-item__currency-container">
+            <LoadingHeartBeat estimateUsed={estimateUsed} />
+            <UserPreferencedCurrencyDisplay
+              type={SECONDARY}
+              value={hexMinimumTransactionFee}
+              hideLabel={Boolean(useNativeCurrencyAsPrimaryCurrency)}
```

Comment:
re: duplicate display value for testnet networks

\- starting a thread here to continue the convo above -

Using testnet networks and whether or not the "Primary Currency" setting is set to the current network currency or fiat, I am seeing the duplicate value in the `display-item` components.
<img width="383" alt="Screen Shot 2023-01-06 at 7 47 37 PM" src="https://user-images.githubusercontent.com/20778143/211022393-5e40b728-181f-4de0-a49b-3e81ee3f39c5.png">

<img width="395" alt="Screen Shot 2023-01-06 at 7 40 44 PM" src="https://user-images.githubusercontent.com/20778143/211022803-55e296f9-45ca-42fd-9bc4-4af6054b8e55.png">

Using Ethereum Mainnet, and whether or not the "Primary Currency" is set, I am seeing the fiat value displayed. This appears correct as mentioned in [above PR comments](https://github.com/MetaMask/metamask-extension/pull/16772#issuecomment-1363184217).
<img width="390" alt="Screen Shot 2023-01-06 at 7 43 26 PM" src="https://user-images.githubusercontent.com/20778143/211026765-d28c858c-82f4-4be3-81cb-8f1f6a3de96d.png">
<img width="391" alt="Screen Shot 2023-01-06 at 7 43 52 PM" src="https://user-images.githubusercontent.com/20778143/211026722-12fdbd7c-3610-471a-a9ac-a3de320c0660.png">


I'm thinking we can exclude the currency rate display for testnet currencies. Thoughts? @NiranjanaBinoy @adonesky1 @jpuri

Reply:
Hey @NiranjanaBinoy, 

Sorry, I mixed up the relevant setting configs in my first comment. I think we can ignore `showFiatInTestnets`. Instead, the two relevant values are `showFiatConversionInTestnets` and `useCurrencyRateCheck`. I can reproduce the issue when `useCurrencyRateCheck` is true and `showFiatConversionInTestnets` is false. 

This recording has the reproduced example. It shows three states: 
| State in video | `useCurrencyRateCheck` | `showFiatConversionInTestnets` | Result |
| :---: | :---: | :---: | :---: |
| 1 | false | false |  ✅ |
| 2 | true | false |  duplicate value shown |
| 3 | true | true |  ✅ |


https://user-images.githubusercontent.com/20778143/211341448-6aeac8c2-c35a-43be-9718-b31ea7e25c02.mov

Are you able to reproduce it with these states?

Reply:
I see and confirmed this is true. Thanks for clarifying! 
I created an issue ticket to remove the redundant value shown https://github.com/MetaMask/metamask-extension/issues/17118